demo/README.md

```diff
@@ -49,7 +49,7 @@ make consume
 
 This scenario uses `S3Storage` with the real AWS S3 as the remote storage.
 
-Please note that the in-memory cache `io.aiven.kafka.tieredstorage.chunkmanager.cache.InMemoryChunkCache` is used in this scenario. This is necessary to mitigate a now-present bug in Kafka with slow and excessive read from S3.
+Please note that the in-memory cache `io.aiven.kafka.tieredstorage.fetch.cache.InMemoryChunkCache` is used in this scenario. This is necessary to mitigate a now-present bug in Kafka with slow and excessive read from S3.
```

Comment:
I guess this line is inaccurate in both sentences:
1. We switched to the disk cache in demos.
2. The second issue is not what the caches are against nowadays :) 

Let's remove it altogether?